spec/feature_spec/user_stories_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'takeaway'
+require 'menu'
+require 'order'
+require 'phone'
+
+describe :UserStories do
+  let(:menu) { Menu.new }
+  let(:order_klass) { Order }
+  let(:phone) { Phone.new }
+  let(:takeaway) { Takeaway.new menu, order_klass, phone }
+
+  before do
+    allow(phone).to receive(:send_text)
+  end
+  
+  #As a customer
+  #So that I can check if I want to order something
+  #I would like to see a list of dishes with prices
+  it 'customer can access menu' do
+    expect(takeaway.menu)
+      .to eq({ 'Spring Rolls' => 2.99, 'Crispy Duck' => 6.99})
```

Comment:
Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)